packages/antd/src/routing/MenuAndContentAnt.tsx

```diff
@@ -9,13 +9,9 @@ export type UIProps<DataType> = NavMenuProps<DataType> & LocationDependentAreaPr
  * This widget shows an Ant Nav Menu bar, and the corresponding content,
  * based on the state-space and the location.
  */
-export class MenuAndContentAnt<DataType> extends React.Component<UIProps<DataType>> {
-    public render() {
-        return (
-            <div style={{ width: '100%' }}>
-                <NavMenuBarAnt {...this.props} />
-                <LocationDependentArea {...this.props} />
-            </div>
-        );
-    }
-}
+export const MenuAndContentAnt = (props: UIProps<unknown>) => (
```

Comment:
ESLint is complaining about the frequent usage of `any`.  (See [here](https://typescript-eslint.io/rules/no-explicit-any/).) I have downgraded the severity from error to warning, but still, I'm trying to take it into account.

As far as I can understand, the rationale is that when I don't want to do anything with a value, so I don't care about it, then we should use `unknown` instead of `any`, which is more strict, doesn't allow random assignment to other types, so it is less likely to introduce problems that the TS compiler can't catch. Since we don't want to use any of those values in this method, this doesn't cause us any difficulty.